js/views/settingsVw.js

```diff
@@ -0,0 +1,290 @@
+var __ = require('underscore'),
+    Backbone = require('backbone'),
+    $ = require('jquery'),
+    userProfileModel = require('../models/userProfileMd'),
+    loadTemplate = require('../utils/loadTemplate'),
+    timezonesModel = require('../models/timezonesMd'),
+    languagesModel = require('../models/languagesMd.js'),
+    personListView = require('./userListVw'),
+    countriesModel = require('../models/countriesMd');
+
+module.exports = Backbone.View.extend({
+
+  className: "settingsPage",
+
+  events: {
+    'click .js-generalTab': 'generalClick',
+    'click .js-shippingTab': 'shippingClick',
+    'click .js-storeTab': 'storeClick',
+    'click .js-blockedTab': 'blockedClick',
+    'click .js-advancedTab': 'advancedClick',
+    'click .js-cancelSettings': 'cancelClick',
+    'click .js-saveSettings': 'saveClick',
+    'blur input': 'validateInput',
+    'blur textarea': 'validateInput'
+  },
+
+  initialize: function(options){
+    var self = this;
+    this.options = options || {};
+    this.userProfile = new userProfileModel();
+    this.userProfile.urlRoot = options.userModel.get('server_url') + "profile";
+    this.model = new Backbone.Model();
+    this.pageID = "";
+    this.subViews = [];
+    this.userProfile.fetch({
+      data: $.param({'id': this.pageID}),
+      success: function(model){
+        self.model.set({user: self.options.userModel.toJSON(), page: model.toJSON()});
+        self.render();
+      },
+      error: function(model, response){
+        self.showErrorModal("User Not Found", "Information for user "+options.userID+" cannot be loaded. They may have gone offline.");
```

Comment:
If you call the showErrorModal in initialize, but define the errorModal element in render, this error message won't work because showErrorModal won't know what the errorModal is yet. 

Change this to an alert, or see if you can get this.errorModal = $('.js-messageModal'); to work in initialize.